workshops/templates/workshops/event.html

```diff
@@ -248,8 +248,6 @@ <h4 id="newtodo">New TODO</h4>
 <script type="text/javascript">
   var gmap = null;
 
-  $('#error_event_url').popover({placement: "left", html: true});
-  $('#error_closest_instructors').popover({placement: "left", html: true});
```

Comment:
Because of https://github.com/swcarpentry/amy/pull/871/files#diff-f460535dbd316f187c28498e625394abR75:
```javascript
/*
  Enable Bootstrap Popovers by default.

  Example usage:

  <span class="btn btn-primary"
        data-toggle="popover"
        data-content="Content of a popup">Hover, focus or click me!</span>
  */
  $('[data-toggle="popover"]').popover({placement: "auto", trigger: "click hover focus"});
```

This piece of JS is loaded on all pages.